.readthedocs.yml

```
@@ -0,0 +1,13 @@
+version: 2
+
+python:
+  version: 3
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

The extra is named just ``doc``, not ``docs``.
```suggestion
        - doc
```